_guides/general/useful-information/Clean-laptop.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: wiki
+title: How to clean your Laptop
+description: "Specifying the methods to clean a laptop without damaging it"
+parent: Useful Information
+grand_parent: General
+image: https://media.discordapp.net/attachments/852210156875153408/955908527724068894/unknown.png
+contributors: ['M164'] 
+---
+
+# How to clean your Legion
+
+## Cleaning the fans
+
+1. Buy a can with compressed air made for cleaning electronics. This kind doesn't contain oil.
+2. Best to remove the back cover, so the dust can be properly pushed out of the laptop.
+3. When cleaning, prevent the fans from spinning. Can use a finger to very gently hold it in place. Holding them in place is to prevent the fans from spinning too fast, which could damage them.
+4. Don't hold the bottle upside to prevent liquid leaking form the can.
+5. Do it outside. The dust cloud that will get released from the laptop will be surreal. Even if it may seem like there is not much dust, the compressed air will push every last bit of it, from every laptop part in range, into the air.
```

Comment:
```suggestion
## Cleaning the fans

1. Buy a can with compressed air made for cleaning electronics. This kind doesn't contain oil.
2. Best to remove the back cover, so the dust can be properly pushed out of the laptop.
3. When cleaning, prevent the fans from spinning. Can use a finger to very gently hold them in place. Holding them in place is to prevent the fans from spinning too fast, which could damage them. Don't press too hard, no need for it. And don't touch the blades of the fans.
4. Don't hold the bottle upside down to prevent liquid leaking form the can. However should the air come out in a liquid form, it will quickly evaporate. 
5. Keep a distance of about 10cm / 4inch between the nozzle and the surface you want to clean. Especially be careful around the fan blades. Don't put the nozzle directly against the blades, as that could damage them.
7. Use quick short bursts. As the air gets released, the temperature of the air in the can will change, affecting the pressure. Using short bursts allows the temperature to return to the normal.
8. Do it outside or in a well ventilated area. The dust cloud that will get released from the laptop will be surreal. Even if it may seem like there is not much dust, the compressed air will push every last bit of it, from every laptop part in range, into the air. Furthermore, the air in the can will most likely be methane, which you do not want to have in excess in your room.
```